src/garage/np/_functions.py

```diff
@@ -41,6 +42,8 @@ def obtain_evaluation_episodes(policy,
         max_episode_length (int): Maximum episode length. The episode will
             truncated when length of episode reaches max_episode_length.
         num_eps (int): Number of episodes.
+        deterministic (bool): Whether the a deterministic approach is used
```

Comment:
Typo, please correct before merging. 

Could be better phrased as: when using stochastic policies, whether random true if deterministic sampling (taking the mean of the distribution outputted by the stochastic policy) is done and false if random sampling is done (sampling from the outputted distribution).